src/static-pages/index.jsx

```diff
@@ -16,9 +16,10 @@ import IntroInformation from '$/components/IntroInformation';
 import logo from '$/images/sticky-logo-text.svg';
 
 const Index = ({ data }) => {
-  const mainPartner = data.allContentfulPartner.edges.filter(
+  const mainPartners = data.allContentfulPartner.edges.filter(
     edge => edge.node.isMainPartner
-  )[0].node;
+  );
+  const mainPartner = mainPartners.length > 0 ? mainPartners[0].node : null;
```

Comment:
```js
const mainPartner = data.allContentfulPartner.edges.find(
  edge => edge.node.isMainPartner
)?.node;
```

Reply:
You're totally right about these, but we are not too worried about code quality since the website will be reworked from the ground up this year anyhow.